core/ui/i18n/locales/hr.ts

```
@@ -10,7 +10,12 @@ export const hr = {
   onboarding_description_4:
     'Vultisig je novčanik bez seed fraza. Umjesto toga koristi lako upravljive dijelove trezora. Svaki uređaj ima svoj jedinstveni dio koji treba zasebno sigurnosno kopirati.',
   search_field_placeholder: 'Pretraži..',
+  incorrect_passcode: 'Netočan pristupni kod',
+  passcodes_do_not_match: 'Kodovi se ne podudaraju',
   tx_hash: 'Hash transakcije',
```

Biome (1.9.4)

[error] This property value named passcodes_do_not_match is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named passcodes_do_not_match


(lint/suspicious/noDuplicateObjectKeys)